packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -110,6 +110,12 @@ export function createLodestarMetrics(
         help: "Current count of pending items in reqRespBridgeReqCaller data structure",
       }),
     },
+    networkWorkerWireEventsOnMainThreadLatencySec: register.histogram<"eventName">({
+      name: "lodestar_network_worker_wire_events_on_main_thread_latency",
+      help: "Latency to transmit network events to main thread across worker port",
```

Comment:
The metric name does not follow the [recommended naming convention](https://prometheus.io/docs/practices/naming/#metric-names). In my opinion, no need to suffix the variable name (since we use seconds everywhere) but the metric name on the other hand must be suffixed.

```suggestion
    networkWorkerWireEventsOnMainThreadLatency: register.histogram<"eventName">({
      name: "lodestar_network_worker_wire_events_on_main_thread_latency_seconds",
      help: "Latency in seconds to transmit network events to main thread across worker port",
```